tests/regression_tests/test_benchmark_ddpg.py

```diff
@@ -0,0 +1,268 @@
+            baselines_csv = run_baselines(env, seed, baselines_dir)
+
+            garage_csvs.append(garage_csv)
+            baselines_csvs.append(baselines_csv)
+
+        plot(
+            b_csvs=baselines_csvs,
+            g_csvs=garage_csvs,
+            g_x="Epoch",
+            g_y="AverageReturn",
+            b_x="total/epochs",
+            b_y="rollout/return",
+            trails=task["trials"],
+            seeds=seeds,
+            plt_file=plt_file,
+            env_id=env_id)
+
+
+def run_garage(env, seed, log_dir):
+    """
+    Create garage model and training.
+
+    Replace the ddpg with the algorithm you want to run.
+
+    :param env: Environment of the task.
+    :param seed: Random seed for the trail.
+    :param log_dir: Log dir path.
+    :return:
+    """
+    env.seed(seed)
+
+    with tf.Graph().as_default():
+        # Set up params for ddpg
+        action_noise = OUStrategy(env, sigma=params["sigma"])
+
+        actor_net = ContinuousMLPPolicy(
+            env_spec=env,
+            name="Actor",
+            hidden_sizes=params["actor_hidden_sizes"],
+            hidden_nonlinearity=tf.nn.relu,
+            output_nonlinearity=tf.nn.tanh)
+
+        critic_net = ContinuousMLPQFunction(
+            env_spec=env,
+            name="Critic",
+            hidden_sizes=params["critic_hidden_sizes"],
+            hidden_nonlinearity=tf.nn.relu)
+
+        ddpg = DDPG(
+            env,
+            actor=actor_net,
+            critic=critic_net,
+            actor_lr=params["actor_lr"],
+            critic_lr=params["critic_lr"],
+            plot=False,
+            target_update_tau=params["tau"],
+            n_epochs=params["n_epochs"],
+            n_epoch_cycles=params["n_epoch_cycles"],
+            n_rollout_steps=params["n_rollout_steps"],
+            n_train_steps=params["n_train_steps"],
+            discount=params["discount"],
+            replay_buffer_size=params["replay_buffer_size"],
+            min_buffer_size=int(1e4),
+            exploration_strategy=action_noise,
+            actor_optimizer=tf.train.AdamOptimizer,
+            critic_optimizer=tf.train.AdamOptimizer)
+
+        # Set up logger since we are not using run_experiment
+        tabular_log_file = osp.join(log_dir, "progress.csv")
+        tensorboard_log_dir = osp.join(log_dir, "progress")
+        garage_logger.add_tabular_output(tabular_log_file)
+        garage_logger.set_tensorboard_dir(tensorboard_log_dir)
+
+        ddpg.train()
+
+        garage_logger.remove_tabular_output(tabular_log_file)
+
+        return tabular_log_file
+
+
+def run_baselines(env, seed, log_dir):
+    """
+    Create baselines model and training.
+
+    Replace the ddpg and its training with the algorithm you want to run.
+
+    :param env: Environment of the task.
+    :param seed: Random seed for the trail.
+    :param log_dir: Log dir path.
+    :return
+    """
+    rank = MPI.COMM_WORLD.Get_rank()
+    env.seed(seed)
```

Comment:
you need to set the tensorflow seed as well. i think it's `tf.set_random_seed()`